common/djangoapps/entitlements/models.py

```diff
@@ -30,6 +30,8 @@ class CourseEntitlementPolicy(models.Model):
     DEFAULT_EXPIRATION_PERIOD_DAYS = 730
     DEFAULT_REFUND_PERIOD_DAYS = 60
     DEFAULT_REGAIN_PERIOD_DAYS = 14
+    DEFAULT_ENROLLMENT_MODE = CourseMode.VERIFIED
```

Comment:
I am not sure we should default this to a mode to Verified.  We should require an entry when setting the policy.  Defaulting this applies EDX specific logic about modes to the table.

Reply:
I understand that reasoning, but I also think that we were assuming a mode of verified with to be associated with the other defaults, which is why I added the default mode value (so as to have the assumption instead be explicit). Correct me if I'm wrong, though, that the default time period values were assumed to have been associated specifically with entitlements with the mode verified.

Reply:
While this old assumption is true it was also meant to be a Default for all Entitlements and Policies.  So it is more inline with your test for No site, No mode policy.